tests/ci/run_posix_tests.sh

```diff
@@ -16,6 +16,9 @@ build_and_test -DOPENSSL_SMALL=1 -DCMAKE_BUILD_TYPE=Release
 echo "Testing AWS-LC in no asm mode."
 build_and_test -DOPENSSL_NO_ASM=1 -DCMAKE_BUILD_TYPE=Release
 
+echo "Testing AWS-LC with AArch64 P-256 optimizations."
+build_and_test -DOPENSSL_AARCH64_P256=1 -DCMAKE_BUILD_TYPE=Release
+
```

Comment:
The flag `OPENSSL_AARCH64_P256` is arm specific. But this test will also be executed in x86. If this is the purpose, can the flag be renamed?

Reply:
Any updates?

Reply:
That is a correct observation, Bryce. 
* My goal for now was to show that whether the switch was enabled or disabled, the tests run on both platforms.
* I prefer to keep AARCH64 in the flag name as it only pertains to the code run on that platform.
* In subsequent PR, the flag will likely be enabled in the code and not as a build flag (as per discussion with Daryl).